sdk/java-sdk-protobuf-testkit/src/main/java/kalix/javasdk/testkit/KalixProxyContainer.java

```diff
@@ -41,7 +41,7 @@ public class KalixProxyContainer extends GenericContainer<KalixProxyContainer> {
   static {
     String customImage = System.getenv("KALIX_TESTKIT_PROXY_IMAGE");
     if (customImage == null) {
-      DEFAULT_PROXY_IMAGE_NAME = DockerImageName.parse(BuildInfo.proxyImage()).withTag(BuildInfo.proxyVersion().replace("-SNAPSHOT", ""));
```

Comment:
I changed it because when working **locally** with a proxy snapshot like: `1.1.7-M1-18-c6b96c08-SNAPSHOT` the docker image name is actually without the `-SNAPSHOT` postfix, only  `1.1.7-M1-18-c6b96c08`, so it was necessary to change evn value each time. For the ubi app test, the docker image name contains the snapshot postfix.
```
docker image tag gcr.io/kalix-public/kalix-proxy:latest gcr.io/kalix-public/kalix-proxy:${PROXY_SNAPSHOT_VERSION}
```
Since this is blocking, let's merge it, but it will be nice to somehow align the proxy docker image names, e.g. publishing locally with a snapshot postfix. I could give it a try in a separate PR.

Reply:
We can overwrite the image using env var `KALIX_TESTKIT_PROXY_IMAGE`. Since it's an edge case, it might be simpler to use this workaround instead.

Reply:
yeah I see now. I don't have a strong opinion about it to be honest.